src/moh_data/main.py

```diff
@@ -230,6 +250,8 @@ def export_data(data, filename):
         print('Cumulative confirmed cases on day 41 = ', run_data.get_cumulative_confirmed_cases_on_day(40))
         print('Cumulative probable cases on day 41 = ', run_data.get_cumulative_probable_cases_on_day(40))
         print('Cumulative total cases on day 41 = ', run_data.get_cumulative_total_cases_on_day(40))
+        print('Cumulative recovered cases on day 80 = ', run_data.get_cumulative_recovered_cases_on(80))
+        print('Cumulative dead cases on day 80 = ', run_data.get_cumulative_dead_cases_on(80))
```

Comment:
The day reference is wrong. I mean that day 80 for JHU corresponds to day 43 for NZ (day 0 is 28 February, I think).